P8-New-Note-Controller/content.md

```diff
@@ -245,7 +245,7 @@ As it stands we have just added support for our `Save Action`.
 2. Realm allows for advanced sorting and query functionality for it's stored objets, preivously we just grabbed all Note objects without any regard for order, this change makes it more useful 
 and orders by the most recent `modificationDate`.
 
-Before you run the app let's tidy up the `viewDidLoad()` function, previously you added test code to create a new Note everytime the app is run.  Time to tidy this code up now.
+Before you run the app let's tidy up the `viewDidLoad()` function in `NotesViewController`, previously you added test code to create a new Note everytime the app is run.  Time to tidy this code up now.
```

Comment:
Yeah that's true. What I thought could cause some confusion is that the part before it has the following sentence: We need to grab a reference to the outgoing controller, in this case our New Note View Controller, we do this to gain access to the currentNote variable that holds the new Note object.

Not really sure if it would confuse people or not.